### DISCUSSION TITLE
does frb work with safari?

### DISCUSSION THREAD

**Discussion by uditrugman** [fzyzcjy/flutter_rust_bridge#1612] [Q&A]
I'm trying to run flutter_via_create from frb_examples in safari and I getting the below error:

[Error] 
	ae_ (main.dart.js:6353)
	ae0 (main.dart.js:6437)
	$0 (main.dart.js:30611)
	ajl (main.dart.js:5802)
	ajx (main.dart.js:5804)
	$1 (main.dart.js:30044)

is safari support with frb?

**Comment by fzyzcjy** [comment]
Hi, seems that this is the same as https://github.com/fzyzcjy/flutter_rust_bridge/discussions/1611, so I close it
